releases/index.md

```diff
@@ -6,8 +6,19 @@ docs_area: releases
 toc_not_nested: true
 ---
 
+{% comment %}
+NOTE TO WRITERS: This file contains interleaved HTML and Liquid. To ease maintenance and readability
+of this file, block-level HTML is indented in relation to the other HTML, and block-level Liquid is
+indented in relation to the other Liquid. Please try to keep the indentation consistent. Thank you!
+{% endcomment %}
+
 After downloading your desired release, learn how to [install CockroachDB](../{{site.versions["stable"]}}/install-cockroachdb.html). Also be sure to review Cockroach Labs' [Release Support Policy](release-support-policy.html).
 
+The following binaries are not suitable for production environments:
+
+- **Testing** binaries allow you to validate the next major or minor version of CockroachDB while it is in development. A testing release is categorized by its level of maturity, moving from Alpha to Beta to Release Candidate (RC).
```

Comment:
@mwang1026 PTAL at this language and the line below, per @kenliu-crl

Reply:
Overall makes sense. Is it right to assume that we're labeling ARM builds as `Experimental`?

One piece of feedback is that we (and this may be recent) *do not guarantee* that you can upgrade from an Alpha/Beta/RC to the actual full release (as in, you might not be able to go from 22.1.X -> 22.2-alpha1 -> 22.2.0. I feel like warrants a callout *somewhere* in our docs, I'm just not sure where. At the very least, I think it should be called out here -- the question is whether it requires add'l callout

Reply:
Yes, it might be more clear if you look at the HTML preview.

Reply:
Is there a way to refresh the preview with the latest changes? It's been a while since I reviewed a docs issue that needed a UX review as well!

When I clicked the preview from the PR description it did stick out to me that it's not super clear that Windows is Experimental (just a parenthetical on the column header of the link to download column).

Thoughts on either
(1) Have the windows election button have a denotation of experimental? Like that row with all the platforms to pick a download from
or
(2) Have the actual link itself say Experimental?

I'd assume it'd work the same way for Windows -- is there a way to generate a preview with ARM64? Might be hard given we're not publishing an ARM build yet

Reply:
I think part of what makes it weird is that Testing and Experimental are NOT mutually exclusive! Like, testing builds are also not qualified for production usage, in a sense. Unfortunately don't have concrete ideas to handle the cognitive dissonance here and OK if the answer is "you're thick, Michael"

Reply:
I'm confused about why you're not seeing the preview. In the latest version, I added a Javascript alert for all Experimental download links (not the SHA256 links). This has been validating by the docs tooling folks, but does it help address your concern?